sass/base/helpers.sass

```diff
@@ -46,12 +46,66 @@
 
```

Comment:
oh nice, yea much nicer! If there is interest in the PR I'll be happy to change it to that for sure!